Core/GraphQL/GraphQLManager.swift

```diff
@@ -60,6 +60,10 @@ extension GraphQLManager: GraphQLManageable {
         ) { result in
             switch result {
             case .success(let graphQLResult):
+                if let errors = graphQLResult.errors,
+                   let firstError = errors.first {
```

Comment:
I just realised you can also write it in one line `let firstError = graphQLResult.errors?.first` 😄  Up to you if you want to improve it

Reply:
Oh yes, I didn't realise it earlier. 😆  It's much better, thanks!